src/main/steps/c100-rebuild/c100sequence.ts

```diff
@@ -560,7 +562,20 @@ export const C100Sequence: Step[] = [
   {
     url: C100_APPLICANT_ADD_APPLICANTS,
     showInSection: Sections.C100,
-    getNextStep: () => C100_APPLICANT_ADD_APPLICANTS_CONFIDENTIALITY_DETAILS_KNOW,
+    getNextStep: () => C100_APPLICANT_STAYING_IN_REFUGE,
```

Comment:
Which sequence will take the user to C100_APPLICANT_ADD_APPLICANTS_CONFIDENTIALITY_DETAILS_KNOW?

Reply:
it is geeting used in src/main/steps/c100-rebuild/applicant/navigationController.ts